jobs/config.json

```diff
@@ -3862,6 +3862,31 @@
       "sig-apps"
     ]
   },
+  "ci-kubernetes-e2e-gce-ubuntu1-k8sdev-default": {
+    "_comment": "AUTO-GENERATED by experiment/generate_tests.py - DO NOT EDIT.",
+    "args": [
+      "--check-leaked-resources=false",
+      "--cluster=test-118970819f",
+      "--env=CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1",
+      "--extract=ci/latest",
+      "--gcp-node-image=CUSTOM",
+      "--gcp-project=ubuntu-os-gke-cloud-dev-tests",
+      "--gcp-zone=us-central1-f",
+      "--ginkgo-parallel=30",
+      "--image-family=ubuntu-gke-1604-lts-1",
+      "--image-project=ubuntu-os-gke-cloud-devel",
+      "--provider=gce",
```

Comment:
i think this needs to be gke

Reply:
this job is intentionally generated with GCE config, the job below is GKE. not sure if that makes sense given the context though..

Reply:
Sorry, I should have clarified this. This is intentional. I'd like to create two jobs (gce and gke) and make sure the same `gcp-node-image` works on both platforms.